src/chocolatey/infrastructure.app/services/INugetService.cs

```diff
@@ -16,9 +16,11 @@
 
 namespace chocolatey.infrastructure.app.services
 {
-    using configuration;
+	using chocolatey.infrastructure.results;
+	using configuration;
+	using System.Collections.Concurrent;

```

Comment:
Minor nitpick...  System namespaces should appear first, before any other namespaces.

Reply:
I thought I was nitpicky ;)